### PR TITLE
New chain logos for V3

### PR DESCRIPTION
**Pull request by aroralanuk**
Added black and color SVGs of Scroll, Polygon zkEVM and Base chains.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,7 +75,7 @@
     "url": "https://github.com/hyperlane-xyz/hyperlane-widgets"
   },
   "scripts": {
-    "build": "yarn build:svg && yarn build:cjs && yarn build:esm && yarn build:css",
```
Comment (jmrossy) [inline package.json:78]: Which logo is breaking during the `svgr` step? I'll take a look

**Review by jmrossy — COMMENTED** [review]
Looks good! I'd just like to sort out the svgr issue you saw before we merge

File: package.json (hunk 1 above)
Comment (aroralanuk) [reply package.json:78]: All the color ones - scroll, polygonzkevm, base

**Comment by jmrossy** [comment]
Superseded by 3.1.4 update last week

**Pull request closed without merging**